### PR TITLE
Added improved error handling of the useParcelMapLayer service | psp-7416

### PR DESCRIPTION
**Pull request by FuriousLlama**

### CODE REVIEW

File: source/frontend/src/components/common/mapFSM/useLocationFeatureLoader.tsx [hunk 1]
```diff
@@ -48,52 +48,49 @@ const useLocationFeatureLoader = () => {
...
+      }
+
+      const municipalityFeature = await adminLegalBoundaryLayerServiceFindOneMunicipality(latLng);
 
-        result.parcelFeature = parcelFeature ?? null;
-        result.regionFeature = regionFeature ?? null;
-        result.districtFeature = districtFeature ?? null;
-        result.municipalityFeature = municipalityFeature ?? null;
-      } finally {
-        // TODO: Remove once the try above is deemed no longer necessary.
```
Comment (FuriousLlama) [inline source/frontend/src/components/common/mapFSM/useLocationFeatureLoader.tsx:94]: this is the only real change, the rest is the try wrapping removal being marked as a change

File: source/frontend/src/hooks/layer-api/useWfsLayer.ts [hunk 2]
```diff
@@ -67,6 +67,7 @@ export const useWfsLayer = (
       },
       [layerOptions, url],
     ),
+    throwError: true,
```
Comment (FuriousLlama) [inline source/frontend/src/hooks/layer-api/useWfsLayer.ts:70]: the usewfs layer will bubble up the errors now

File: source/frontend/src/hooks/repositories/mapLayer/useParcelMapLayer.ts [hunk 3]
```diff
@@ -24,61 +25,85 @@ export const useParcelMapLayer = () => {
 
   const { execute: getAllFeatures, loading: getAllFeaturesLoading } = getAllFeaturesWrapper;
 
+  const handleError = useCallback((error: unknown) => {
+    toast.error('Unable to contact Parcel Map');
+  }, []);
```
Comment (FuriousLlama) [inline source/frontend/src/hooks/repositories/mapLayer/useParcelMapLayer.ts:30]: at some point in the future we should improve the network slice to maintain the state of the requests and use a component that monitors it to show or not show errors. For now this is just using the regular toasts as the other services

File: source/frontend/src/hooks/layer-api/useWfsLayer.ts (hunk 2 above)
Comment (asanchezr) [reply source/frontend/src/hooks/layer-api/useWfsLayer.ts:70]: Did you check all usages of `useWfsLayer` to make sure they handle errors correctly?

**Review by asanchezr — APPROVED** [review]

**Pull request closed without merging**